UX changes to pending payment page

pauljickling opened the pull request:
### What was wrong?

UX could be improved as described in issue #98

### How was it fixed?

- Added icons for QR Code, web3modal, and uniswap
- Added toggle button for QR Code (color based on bootstrap/Pretix config)
- Minor text improvements
- Changed links to buttons for Web3Modal and Uniswap (colors based on brand identity)
- Added some additional spacing between options
- Changed parentheticals to single unordered list items
- Generally tried to create a more consistent visual order
- Added a light gray background color to the address for manual payment option
- Made address text-wrap for smaller displays

Note that I did not attempt to change the currency display at the top per the reasoning from @davesque.

Screenshot:

<img width="1150" alt="pending" src="https://user-images.githubusercontent.com/18646852/82514251-1daa6900-9aca-11ea-951f-9270f341729e.png">


#### Cute Animal Picture

![deer](https://user-images.githubusercontent.com/18646852/82514145-c906ee00-9ac9-11ea-9b26-ecadada2419b.jpg)

skylarweaver commented on pretix_eth/templates/pretix_eth/pending.html:
```diff
@@ -7,43 +7,44 @@
+      </a>
+      <ul class="payment-info">
+        <li>{% trans "Allows you to pay with any uniswap supported token." %}
+          <strong>{% trans "Please do not change the output amount!" %}</strong>
```
```suggestion
          <strong>{% trans "Please do not change the output amount or output currency!" %}</strong>
```

ligi replied:
:+1:

skylarweaver commented on pretix_eth/templates/pretix_eth/pending.html:
```diff
@@ -7,43 +7,44 @@
+        <button class="btn btn-primary" id="web3modal_btn">{% trans "Web3Modal" %}</button>
+      </a>
+      <ul class="payment-info">
+        <li>{% trans "Allows you to pay with WalletConnect and MetaMask" %}</li>
```
```suggestion
        <li>{% trans "Allows you to pay with WalletConnect or MetaMask." %}</li>
```

pauljickling commented:
@skylarweaver your proposed changes sound good to me. What do you think @ligi ?

ligi commented on pretix_eth/templates/pretix_eth/pending.html:
```diff
@@ -7,43 +7,44 @@
+      <img src={% static "icons/qr_code.png" %} id="qr_icon" />
+      <button class="btn btn-primary" id="btn-qr">{% trans "Toggle QR Code" %}</button>
+      <ul class="payment-info">
+        <li>{% blocktrans trimmed %}Via <a href="https://eips.ethereum.org/EIPS/eip-681">ERC-681</a> QR-Code{% endblocktrans %}</li>
```
not sure if we need to mention "exact amount to the correct address." - sounds a bit weird to me

ligi commented:
one suggestion (af840a9) I took - with the other one I am not yet sure and commented

pauljickling commented:
Made some additional changes to the manual pay option. Included it on the top per Skylar's suggestion with the additional text provided below. I also added a copy to clipboard icon that the user can click so that it automatically copies the address to the clipboard. Haven't tested that on mobile browsers obviously, but it was working on Chromium, Firefox, and Safari desktop browsers. Of course users can always just select text and ctrl + c as normal, just a quality of life enhancement.

Other minor thing is I renamed the CSS file from pending.css to eth_plugin.css with the idea that if there are any other CSS changes that we want to make to the plugin we should just have a single file that manages that for all pages.

Demo (also includes the QR code toggle since I forgot to include that last time):

![Screen Recording 2020-05-22 at 1 27 13 PM](https://user-images.githubusercontent.com/18646852/82708080-46e60900-9c32-11ea-85b1-383bdd455441.gif)

ligi commented:
looks nice - @kvbik can you deploy this one to staging?
